tests/unit/type/set.tcl

```diff
@@ -241,6 +241,11 @@ start_server {
             assert_equal $expected [lsort [r sunion set1{t} set2{t}]]
         }
 
+        test "SUNION with three same sets - $type" {
+            set expected [lsort -uniq "[r smembers set1{t}] [r smembers set1{t}] [r smembers set1{t}]"]
+            assert_equal $expected [lsort [r sunion set1{t} set1{t} set1{t}]]
+        }
```

Comment:
maybe i missed something you mentioned in your comments, but our change is about SDIFF (not SUNION), let's make sure to test it too.

Reply:
I would like to confirm what you mean.
Do you mean we need to add a test: `sdiff with three same sets`? Test that they always have an empty result?
If so, do we need to add this test for `sinter`? Or just test the `sdiff`?

Reply:
while we're at it, let's test all 3.
could be just one line per command, all in one test.